libraries/provider_install_source.rb

```diff
@@ -0,0 +1,68 @@
+class Chef
+  # Chef Provider for installing or removing Elasticsearch from package or source
```

Comment:
The description should read `installing or removing Elasticsearch from the official archive`.

Note that "installing from source" usually means compiling the software -- something we definitely want to provide in the future (ie. `git clone`, `mvn package` and all that).